src/Uno.UI/UI/Xaml/Controls/MenuFlyout/MenuFlyoutPresenter.cs

```diff
@@ -1,13 +1,958 @@
+							var subPresenterBoundsLogical = GetSubPresenterBounds(spSubPresenterAsUIE);
+							var containsSubPresenter = subPresenterBoundsLogical.Contains(clientLogicalPointerPosition);
+
+							if (!containsSubPresenter)
+							{
+								DelayCloseMenuFlyoutSubItem();
+							}
+						}
+					}
+				}
+
+				pArgs.Handled = true;
+			}
+		}
+
+        internal bool GetContainsToggleItems()
+        {
+            return m_containsToggleItems;
+        }
+
+		// Returns true if the ItemsSource contains at least one MenuFlyoutItem with an Icon; false otherwise.
+		internal bool GetContainsIconItems()
+        {
+            return m_containsIconItems;
+        }
+
+		// Returns true if the ItemsSource contains at least one MenuFlyoutItem with an Icon; false otherwise.
+		internal bool GetContainsItemsWithKeyboardAcceleratorText()
+        {
+            return m_containsItemsWithKeyboardAcceleratorText;
+        }
+
+		private Rect GetSubPresenterBounds(UIElement pSubPresenterAsUIE)
+		{
+			var t = pSubPresenterAsUIE.TransformToVisual(null);
+			var r = t.TransformBounds(pSubPresenterAsUIE.LayoutSlotWithMarginsAndAlignments);
+
+			return r;
+		}
+
+		protected override void OnPointerEntered(PointerRoutedEventArgs pArgs)
+		{
+			base.OnPointerEntered(pArgs);
+
+			var handled = pArgs.Handled;
+
+			if (!handled)
+			{
+				Pointer spPointer;
+				spPointer = pArgs.Pointer;
+				var pointerDeviceType = spPointer.PointerDeviceType;
+				if (PointerDeviceType.Mouse == pointerDeviceType && m_isSubPresenter)
+				{
+					CancelCloseMenuFlyoutSubItem();
+					var owner = (this as IMenuPresenter).Owner;
+
+					if (owner != null)
+					{
+						var parentSubItem = owner as MenuFlyoutSubItem;
+						if (parentSubItem != null)
+						{
+							var presenter = parentSubItem.GetParentMenuFlyoutPresenter();
+							if (presenter != null)
+							{
+								// When the mouse enters a MenuFlyoutPresenter that is a sub menu
+								// we have to tell the parent presenter to cancel any plans it had
+								// to close this sub 
+								presenter.CancelCloseMenuFlyoutSubItem();
+							}
+						}
+					}
+				}
+
+				pArgs.Handled = true;
+			}
+		}
+
+		string GetPlainText()
+		{
+			string automationName = null;
+
+			var ownerFlyout = m_wrParentMenuFlyout.Target as DependencyObject;
+
+			if (ownerFlyout != null)
+			{
+				// If an automation name is set on the parent flyout, we'll use that as our plain text.
+				// Otherwise, we'll report the default plain text.
+				automationName = AutomationProperties.GetName(ownerFlyout);
+			}
+
+			if (automationName != null)
+			{
+				return automationName;
+			}
+			else
+			{
+				// UNO TODO
+
+				// If we have no title, we'll fall back to the default implementation,
+				// which retrieves our content as plain text (e.g., if our content is a string,
+				// it returns that; if our content is a TextBlock, it returns its Text value, etc.)
+				// MenuFlyoutPresenterGenerated.GetPlainText(strPlainText);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5205492)